src/core/ColorPicker/ColorSwatch.test.tsx

```diff
@@ -8,11 +8,12 @@ import { ColorSwatch } from './ColorSwatch';
 import { ColorValue } from '../utils';
 
 it('should render in its most basic state with ColorValue', () => {
-  const { container } = render(<ColorSwatch color={'#9BA5AF'} />);
-
+  const { container } = render(
+    <ColorSwatch color={ColorValue.create('#9BA5AF')} />,
```

Comment:
why not use the same value here as the one you're testing against? i.e. `'rgb(156, 156, 156)'`